decompressed/gui_file/www/lang/it-it/webui-contentsharing.po

```diff
@@ -28,13 +28,13 @@ msgid "Content Sharing"
 msgstr "Condivisione contenuti"
 
 msgid "Currently there are <strong>%d</strong> connected devices"
-msgstr "Ci sono <strong>%d</strong> dispositivi connessi"
```

Comment:
Queste tre traduzioni precedenti mi sembrano più corrette

Reply:
Seondo em è giusta la modifica proposta.
La stringa "Ci sono" non aggiunge nulla di significativo e allunga inutilmente la stringa. 
""Ci sono" porta in secondo piano l'info principale "x device connessi".
Inoltre in altre pagina della stessa GUI lo standard è x device xxxx